src/implementation.cpp

```diff
@@ -49,6 +50,10 @@ std::string href_from_file(std::string_view input) {
 }
 
 bool can_parse(std::string_view input, const std::string_view* base_input) {
+  if (helpers::is_windows_file_path(input)) {
+    return false;
+  }
```

Comment:
Can you point us to where in the URL spec this is returning false? Chrome returns true...

```
> URL.canParse("file:///C:/path/file.node")
true
```

Reply:
This is actually allowed. https://url.spec.whatwg.org/#file-host-state

If state override is not given and buffer is a [Windows drive letter](https://url.spec.whatwg.org/#windows-drive-letter), [file-invalid-Windows-drive-letter-host](https://url.spec.whatwg.org/#file-invalid-windows-drive-letter-host) [validation error](https://url.spec.whatwg.org/#validation-error), set state to [path state](https://url.spec.whatwg.org/#path-state).

Reply:
If you apply the same change to URL parser, not the can_parse() method, you'll see that it breaks web-platform tests.

Reply:
thank you, instead of rejecting Windows file paths
Is it a healthy method to normalize them with the ```file:///``` protocol ?

Reply:
What the library provides is WHATWG URL support. If the result disagrees with your expectations make sure first to check the WHATWG URL specification. If you disagree with it, you should take your concerns with WHATWG.

We may have bugs but a bug is a disagreement with WHATWG.